### PR TITLE
Add tests for PolygonDrawingInfoDialogFragmentTest

### PR DESCRIPTION
**Pull request by JSunde**
@shobhitagarwal1612  PTAL

### CODE REVIEW

File: gnd/src/test/java/com/google/android/gnd/ui/mapcontainer/PolygonDrawingInfoDialogFragmentTest.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  private PolygonDrawingInfoDialogFragment polygonDrawingInfoDialogFragment;
+
+  private Runnable onClickRunnable;
+  private boolean clicked;
+
+  @Before
+  public void setUp() {
+    super.setUp();
+
+    onClickRunnable = () -> clicked = true;
```
Comment (shobhitagarwal1612) [inline gnd/src/test/java/com/google/android/gnd/ui/mapcontainer/PolygonDrawingInfoDialogFragmentTest.java:51]: Do we need to reset the value of `clicked` in teardown?

File: gnd/src/test/java/com/google/android/gnd/ui/mapcontainer/PolygonDrawingInfoDialogFragmentTest.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    assertThat(dialogView.getVisibility()).isEqualTo(View.VISIBLE);
+    assertThat(
+            ((ConstraintLayout) dialogView.getParent())
+                .findViewById(R.id.polygon_info_image)
+                .getVisibility())
+        .isEqualTo(View.VISIBLE);
+  }
+
+  @Test
+  public void show_dataTypeSelected_correctDataTypeIsPassed() {
```
Comment (shobhitagarwal1612) [inline gnd/src/test/java/com/google/android/gnd/ui/mapcontainer/PolygonDrawingInfoDialogFragmentTest.java:82]: How about `startClicked_dialogIsDismissed`?

**Review by shobhitagarwal1612 — APPROVED** [review]
Minor nits!

File: gnd/src/test/java/com/google/android/gnd/ui/mapcontainer/PolygonDrawingInfoDialogFragmentTest.java (hunk 1 above)
Comment (JSunde) [reply gnd/src/test/java/com/google/android/gnd/ui/mapcontainer/PolygonDrawingInfoDialogFragmentTest.java:52]: Done!

**Review by shobhitagarwal1612 — APPROVED** [review]

**Pull request merged**